src/main/java/com/ecore/roles/model/Role.java

```diff
@@ -0,0 +1,34 @@
+package com.ecore.roles.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@Entity
```

Comment:
Add the @Table annotation with its "name" parameter, in order to don't get a code refactoring in 
the class name leading to a table name mismatch in the database.